src/datahike/api.cljc

```diff
@@ -12,19 +14,34 @@
               [datahike.impl.entity Entity]
               [java.util Date])))
 
+(s/fdef connect
+  :args (s/alt :url (s/cat :store-config string?)
+               :store-only (s/cat :store-config map?)
+               :full (s/cat :store-config map?
+                            :conn-config :connection/config))
+  :ret :datahike/connection)
+
 (def
-  ^{:arglists '([] [config])
-    :doc "Connects to a datahike database via configuration map. For more information on the configuration refer to the [docs](https://github.com/replikativ/datahike/blob/master/doc/config.md).
+  ^{:arglists '([] [store-config] [store-config conn-config])
+    :doc "Connects to a datahike database via store configuration map and an optional connection configuration map. For more information on the configuration refer to the [docs](https://github.com/replikativ/datahike/blob/master/doc/config.md).
 
-          The configuration for a connection is a subset of the Datahike configuration with only the store necessary: `:store`.
+          The store configuration for a connection is a subset of the Datahike configuration with only the store necessary: `:store`.
 
           `:store` defines the backend configuration as hash-map with mandatory key: `:backend` and store dependent keys.
 
           Per default Datahike ships with `:mem` and `:file` backend.
 
-          The default configuration:
+          The default store configuration:
            `{:store {:backend :mem :id \"default\"}}`
 
+          Optionally a connection configuration hash-map with following keys can be added:
+          - `:tx/sync?`: boolean that whether transactions are flushed automatically on `transact`, if false the user needs to manually flush the database via `sync!`
```

Comment:
word missing?

Reply:
I'll have to rebase anyway and fix that with it.